aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs

```diff
@@ -144,60 +126,82 @@ public void OnPlayerDeath(HealthComponent _)
     /// <summary>
     /// When the play button in the main menu is clicked, it wil teleport the player to the game.
     /// </summary>
-    public void PlayGame() => SceneManager.LoadScene(_sceneNameGame);
+    public void PlayGame()
+    {
+        SceneManager.LoadScene(_sceneNameGame);
+    }
 
     /// <summary>
     /// When the Quit To Menu button in the game setting is clicked, it will take you back to the menu.
     /// </summary>
-    public void ToMenu() => SceneManager.LoadScene(_sceneNameStartingMenu);
+    public void ToMenu()
+    {
+        SceneManager.LoadScene(_sceneNameStartingMenu);
+    }
 
     /// <summary>
     /// This button toggles the menu canvas off and the setting canvas on or game settings on and off.
     /// </summary>
     public void OnToggleSettings()
     {
-        if (_preventSettingsToggle) return;
+        _isOnSettings = !_isOnSettings;
 
-        if (_currentSceneName == _sceneNameStartingMenu) // Toggle from menu to menu settings and back.
+        if (_currentSceneName == _sceneNameStartingMenu)
         {
-            _isOnMenuSettings = !_isOnMenuSettings;
-            SettingMenuCanvas.SetActive(_isOnMenuSettings);
-            MenuCanvas.SetActive(!_isOnMenuSettings);
+            MenuCanvas.SetActive(!_isOnSettings);
         }
         else if (_currentSceneName == _sceneNameGame) // Toggle from game to game settings and back.
         {
-            _isOnGameSettings = !_isOnGameSettings;
-            SettingGameCanvas.SetActive(_isOnGameSettings);
-
-            if (_isOnGameSettings) GameManager.Instance.Pause();
+            if (_isOnSettings) GameManager.Instance.Pause();
             else GameManager.Instance.Resume();
 
-            MenuOpenedEvent?.Invoke(_isOnGameSettings);
+            MenuOpenedEvent?.Invoke(_isOnSettings);
         }
+
+        SettingCanvas.SetActive(_isOnSettings);
     }
 
     /// <summary>
-    /// Show the win screen.
+    /// This button toggles the menu canvas off and the help canvas on or help canvas in game on and off.
     /// </summary>
-    public void ShowWinScreen()
+    public void OnToggleHelp()
     {
-        // Mouse visible.
-        MenuOpenedEvent?.Invoke(true);
-
-        // Pause game.
-        GameManager.Instance.Pause();
+        _isOnHelp = !_isOnHelp;
 
-        // Set all off.
-        SetAllCanvasesToInactive();
-        _preventSettingsToggle = true;
+        if (_currentSceneName == _sceneNameStartingMenu) // Toggle from menu to helpscreen and back.
+        {
+            SettingCanvas.SetActive(!_isOnHelp);
+        }
+        else if (_currentSceneName == _sceneNameGame) // Toggle from game to helpscreen and back.
+        {
+            if (_isOnHelp)
+            {
+                GameManager.Instance.Pause();
+                MenuOpenedEvent?.Invoke(_isOnHelp);
+            }
+            else if (!_isOnHelp && !_isOnSettings)
```

Comment:
the `!_isOnHelp` is trivially true, for this branch of the if else statement is not run unless true.

Reply:
That is indeed true! Fixed it.